src/styles/components/_chart_d3.scss

```diff
@@ -0,0 +1,94 @@
+.chart-d3 {
+  position: relative;
+  height: 350px;
+
+  svg {
+    width: 100%;
+  }
```

Comment:
Ensure that the svg uses all available space, so that the chart's calculated width represents the total space this chart can take up and the chart will resize when the window is resized